### ISSUE TITLE
White screen after updating Grasscutter to 1.7.4

### ISSUE THREAD

**Issue by vipotaenko02** [Melledy/Grasscutter#2472]
After updating Grasscutter to version 1.7.4, it no longer loads any player past whitye screen. Loading bar doesn't appear, but UID does (so the server know this player isn't new). Things I tried:
- Updating the resources, as from YuukiPS's GitLab and through Cultivation Download Resources button
- Creating new player, got the first cutscene and then server connection lost when setting up nickname
Grasscutter logs are totally fine, not any resources-related issue (although on 1.7.3 they does appear, it seems like 1.7.4 just don't want to load resources at all). 
![image](https://github.com/Grasscutters/Grasscutter/assets/116735001/f647fb97-9fff-4781-9ce4-4b4fe8383611)

Maybe this is related to an old CPU (I'm running the server on Core 2 Duo without even SSE4.1 instruction set). Grasscutter is running in docker container with all the latest files related to it. Previous version runs totally fine...

**Comment by NotThorny** [comment]
I've pushed a temporary change to the All-in-One builds in Cultivation to check if it resolves this issue. If you could make sure your Grasscutter is closed, then download one of the All-in-Ones (either normal or questing, both should be the same) to get the fix, and try again on one of the "broken" accounts and on a new one as well. 

Please make sure to not use any of the other buttons in the download menu as they will overwrite the attempted fix.

**Comment by vipotaenko02** [comment]
Updated, trying to test now. Straight I get one file that can't be loaded cause it's not present in resources:
10:08:08 <ERROR:TsvUtils> Error loading file './resources/ExcelBinOutput/DungeonLevelEntityConfigData.tsj' - File does not exist. You are missing resources. Note that this file may exist in JSON, TSV, or TSJ format, any of which are suitable.

**Comment by vipotaenko02** [comment]
And it is still stuck on the white screen. For the reference, the version I'm running now is 1.7.4-a00df54bd. If that's the one you recently pushed, then yeah, that bug remains

**Comment by NotThorny** [comment]
Have you tried already with the 1.7.4 jar from releases, or only 1.7.4 from Cultivation AIOs (prior to the one I asked you to try)? Just to try to narrow down if it's specifically due to a commit between 1.7.3 and .4, and not one of the fixes from the AIO.

If you haven't tried with the 1.7.4 jar from releases yet, give that a go and see if the result is the same.

**Comment by vipotaenko02** [comment]
Yeah, my first try in updating the server was through downloading the latest release from github, as well as I've tried to download several different builds from Cultivation's AIO, nothing worked

**Comment by NotThorny** [comment]
I believe I've identified the correct issue this time. If you could download one of the All-in-Ones from Cultivation again and try to login now, it should be resolved.

**Comment by vipotaenko02** [comment]
> I believe I've identified the correct issue this time. If you could download one of the All-in-Ones from Cultivation again and try to login now, it should be resolved.

Thanks a lot, that actually fixed it. Now it is up and running without any additional issues.
